### PR TITLE
Migration from V1 to V2 budgets

### PR DESCRIPTION
**Pull request by ajaiswals**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/manage/ingestion-volume/ingest-budgets/index.md [hunk 1]
```diff
@@ -44,11 +44,13 @@ There are two versions of ingest budgets:
...
-:::important
-You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata based. V2 budgets are manageable with the Ingest Budget Management API **V2**.
-:::
 
-Once a **V1** budget is given a **scope** and changed to **V2** it is permanent and can not be reversed.
+## Migrating V1 budgets to V2
+
+* You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata based.
+* V2 budgets are manageable with the Ingest Budget Management API **V2**.
+* V1 Budgets will be deprecated on September 16th 2024.
```
Comment (kimsauce) [inline docs/manage/ingestion-volume/ingest-budgets/index.md:52]:
```suggestion
* You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata-based.
* V2 budgets are manageable with the Ingest Budget Management API **V2**.
* V1 budgets will be deprecated on September 16, 2024.
```

**Review by kimsauce — APPROVED** [review]

File: docs/manage/ingestion-volume/ingest-budgets/index.md [hunk 2]
```diff
@@ -44,11 +44,13 @@ There are two versions of ingest budgets:
...
 
-:::important
-You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata based. V2 budgets are manageable with the Ingest Budget Management API **V2**.
-:::
 
-Once a **V1** budget is given a **scope** and changed to **V2** it is permanent and can not be reversed.
+## Migrating V1 budgets to V2
+
+* You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata-based.
+* V2 budgets are manageable with the Ingest Budget Management API **V2**.
```
Comment (kimsauce) [inline docs/manage/ingestion-volume/ingest-budgets/index.md:51]:
```suggestion
* V2 budgets are manageable with the [Ingest Budget Management API **V2**](/docs/api/ingest-budget-v2).
```

File: docs/manage/ingestion-volume/ingest-budgets/index.md [hunk 3]
```diff
@@ -44,11 +44,13 @@ There are two versions of ingest budgets:
...
-:::important
-You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata based. V2 budgets are manageable with the Ingest Budget Management API **V2**.
-:::
 
-Once a **V1** budget is given a **scope** and changed to **V2** it is permanent and can not be reversed.
+## Migrating V1 budgets to V2
+
+* You can edit an existing **V1** budget by providing a **scope** to change it to a **V2** budget that is metadata-based.
+* V2 budgets are manageable with the [Ingest Budget Management **V2** API](/docs/api/ingest-budget-v2).
+* V1 budgets will be deprecated on September 16, 2024.
```
Comment (kimsauce) [inline docs/manage/ingestion-volume/ingest-budgets/index.md:52]: @ajaiswals will all users need to migrate from v1 to v2 by this date?

**Comment by kimsauce** [comment]
@ajaiswals I've made some edits - can you please take a look?

File: docs/manage/ingestion-volume/ingest-budgets/index.md (hunk 3 above)
Comment (ajaiswals) [reply docs/manage/ingestion-volume/ingest-budgets/index.md:52]: @kimsauce - yes and we are starting to sending notifications to customer that we have V1 already.

**Review by ajaiswals — COMMENTED** [review]
@kimsauce These changes look good.

**Comment by kimsauce** [comment]
@ajaiswals thank you for the quick response!

File: docs/manage/ingestion-volume/ingest-budgets/index.md (hunk 3 above)
Comment (kimsauce) [reply docs/manage/ingestion-volume/ingest-budgets/index.md:52]: Is this related to [Ingest Budgets for OT collector](https://drive.google.com/file/d/1lBeb3Y8DVtY7tudKhQBV-1WwsUtRcs2w/view)? @ajaiswals

**Pull request merged**